### PR TITLE
Reader: style Gutenberg latest post block

### PR DESCRIPTION
**Pull request by tophertoy**
#### Changes proposed in this Pull Request

This PR deals with just the latest post block from issue: #43595 
Working from: https://made4testing0318.blog/2020/04/16/latest-posts-block/

#### Testing instructions

Taken from: http://calypso.localhost:3000/read/blogs/159889361/posts/1003

Before:
Screenshot 2020-07-21 12 49 41

After:

Screenshot 2020-08-06 16 07 22

### CODE REVIEW

**Comment by tophertoy** [comment]
@bluefuton thoughts so far?

File: client/blocks/reader-full-post/_content.scss [hunk 1]
```diff
@@ -286,6 +286,21 @@
 	.wp-block-embed__wrapper iframe[src*='icloud.com'] {
 		min-height: 300px;
 	}
+
+	.wp-block-latest-posts__list {
+		margin: 0px;
```
Comment (bluefuton) [inline client/blocks/reader-full-post/_content.scss:291]: No units required for zero :)

File: client/blocks/reader-full-post/_content.scss [hunk 2]
```diff
@@ -286,6 +286,21 @@
...
+		list-style: none;
+		margin-bottom: 4rem;
```
Comment (bluefuton) [inline client/blocks/reader-full-post/_content.scss:293]: Shall we try reducing the top and bottom margins a bit? I'm happy with it having similar spacing to a normal paragraph.

Comment (tophertoy) [reply client/blocks/reader-full-post/_content.scss]: Apologies, I wasn't quite ready for code review. Resolved now.

Comment (tophertoy) [reply client/blocks/reader-full-post/_content.scss]: Ok. Should be roughly the same now:

**Comment by bluefuton** [comment]
Much improved 👍 

Can we investigate what happens with featured images? I notice the penguins on this post are misaligned:

http://calypso.localhost:3000/read/blogs/159889361/posts/760

Screen Shot 2020-08-06 at 16 26 15

**Comment by tophertoy** [comment]
@bluefuton yes I can do that.

I noticed it's in production too: https://wordpress.com/read/blogs/159889361/posts/760

Screen Shot 2020-08-10 at 3 29 02 PM

What is the preferred Automattic process to proceed with a production fix?

 Do you normally check if someone has raised an issue first and see if anyone is working on it?

**Comment by bluefuton** [comment]
> What is the preferred Automattic process to proceed with a production fix?

@tophertoy it's usually best to check for similar issues in the repo first, and then go ahead and create a PR if it isn't actively being worked on. In the case of Reader, you're generally good to go, because it's only receiving basic maintenance.

Sorry if I wasn't clear - I think we should adjust the display of images specifically in the new posts block so they appear consistently. In particular:
* the image should not be taller than the current row
* alignment of the image should be consistent on every row

**Comment by sixhours** [comment]
@tophertoy a much-belated thank you for working on this! Some new styles have been added since this PR opened; the current state looks like this:

Screen Shot 2021-05-13 at 4 31 40 PM

Not great, but not as bad as before. With this PR, we'd lose the bullet points, and I hesitate to do that because they provide some necessary visual separation between each post:

Screen Shot 2021-05-13 at 4 31 48 PM

I'm going to close this out, but please feel free to start a new PR with additional visual improvements based on the latest `trunk`. Apologies it took so long for us to get back to this!

**Pull request closed without merging**